Bump soversion for 1.0.2 series where >=1.0.2g as this release breaks ABI.

--- openssl-1.0.2g/crypto/opensslv.h
+++ openssl-1.0.2g/crypto/opensslv.h
@@ -88,7 +88,7 @@
  * should only keep the versions that are binary compatible with the current.
  */
 # define SHLIB_VERSION_HISTORY ""
-# define SHLIB_VERSION_NUMBER "1.0.0"
+# define SHLIB_VERSION_NUMBER "1.0.1"
 
 
 #ifdef  __cplusplus
